docs/api/actions-and-connectors/list.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[actions-and-connectors-api-list]]
+=== List action types API
+++++
+<titleabbrev>List all action types API</titleabbrev>
+++++
+
+Retrieve a list of all {kib} action types.
+
+[[actions-and-connectors-api-list-request]]
+==== Request
+
+`GET <kibana host>:<port>/api/actions/list_action_types`
+
+[[actions-and-connectors-api-list-codes]]
+==== Response code
+
+`200`::
+    Indicates a successful call.
+
+[[actions-and-connectors-api-list-example]]
+==== Example
+
+[source,sh]
+--------------------------------------------------
+$ curl -X GET api/actions/list_action_types
+--------------------------------------------------
+// KIBANA
+
+The API returns the following:
+
+[source,sh]
+--------------------------------------------------
+[
+  {
+    "id": ".email",
+    "name": "Email",
+    "minimumLicenseRequired": "gold",
+    "enabled": false,
+    "enabledInConfig": true,
+    "enabledInLicense": false
+  },
+  {
+    "id": ".index",
+    "name": "Index",
+    "minimumLicenseRequired": "basic",
+    "enabled": true,
+    "enabledInConfig": true,
+    "enabledInLicense": true
+  }
+]
+--------------------------------------------------
+
+Each action type object contains the following properties:
```

Comment:
We typically document properties before the example, and in the following format:

////
Guidelines for parameter documentation
***************************************
* Use a definition list.
* End each definition with a period.
* Include whether the parameter is Optional or Required and the data type.
* Include default values as the last sentence of the first paragraph.
* Include a range of valid values, if applicable.
* If the parameter requires a specific delimiter for multiple values, say so.
* If the parameter supports wildcards, ditto.
***************************************
////
[[sample-api-path-params]]
==== {api-path-parms-title}
////
A list of all the parameters within the path of the endpoint (before the query string (?)).

For example:
`<follower_index>`::
(Required, string) Name of the follower index
////

Reply:
In this case, these are not parameters to the request, rather an explanation of the properties you will receive in the response.

Reply:
I understand now! I would do something like this:

[source,sh]
--------------------------------------------------
[
  {
    "id": ".email", <1> 
    "name": "Email", <2>
    "minimumLicenseRequired": "gold", <3>
    "enabled": false, <4>
    "enabledInConfig": true, <5>
    "enabledInLicense": false <6>
  },
  {
    "id": ".index",
    "name": "Index",
    "minimumLicenseRequired": "basic",
    "enabled": true,
    "enabledInConfig": true,
    "enabledInLicense": true
  }
]
--------------------------------------------------
<1> `id` — The unique ID of the action type.
<2> `name` — The name of the action type.
<3> `minimumLicenseRequired` — The license required to use the action type.
<4> `enabled` — Specifies if the action type is enabled or disabled in {kib}.
<5> `enabledInConfig` — Specifies if the action type is enabled or enabled in the {kib} .yml file.
<6> `enabledInLicense` — Specifies if the action type is enabled or disabled in the license.

Reply:
Updated!